src/modules/orderItem/orderItem.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { OrderItemService } from './orderItem.service';
+import { OrderItemController } from './orderItem.controller';
+import { OrderItemDao } from './orderItem.dao';
+import { OrderItem } from '../../../entities/OrderItem.entity';
+
+@Module({
```

Comment:
As discussed earlier last week, this entire module may be unnecessary, please double-check and only leave what's really needed.

Reply:
@jfernandezdev14 Exactly, probably we only need the DB table (migration), the entity and that's it.

Reply:
👍